api/observability/tracing/ssh/ssh.go

```diff
@@ -45,6 +49,14 @@ const (
 	instrumentationName = "otelssh"
 )
 
+// EnvsReq contains json marshaled key:value pairs sent as the
+// payload for an EnvsRequest.
+type EnvsReq struct {
+	// Envs is a json marshaled map[string]string containing
+	// environment variables.
+	Envs []byte
```

Comment:
Just in case, so it doesn't rely on the field name?

```suggestion
	Envs []byte `json:"envs"`
```